workspace.code-workspace

```diff
@@ -0,0 +1,13 @@
+{
+  "folders": [
+    {
+      "path": "src/FunctionApps/PHDISmartyStreetsTest"
+    },
+    {
+      "path": "src/FunctionApps/PITest_FunctionApp"
+    }
+  ],
+  "settings": {
+    "debug.internalConsoleOptions": "neverOpen"
+  }
+}
```

Comment:
Did you mean to commit this?

Reply:
Yep - I should actually do a brief writeup about this; using a workspace makes it a lot easier to work with and deploy Azure Functions in VSCode as it works better with the Azure extension. I can write this up and add to this PR!

Reply:
(I know Bryan is working on this also, I can make sure we combine notes)